packages/doc/src/components/Clipboard/index.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-undef */
+import React, { useState } from 'react';
+import { Button, Box } from '@gympass/yoga';
+import { string } from 'prop-types';
+
+import { CheckedFull, Copy } from '@gympass/yoga-icons/src';
+
+const Clipboard = ({ copyText }) => {
+  const [isCopied, setIsCopied] = useState(false);
+
+  async function copyTextToClipboard(text) {
+    const textToCopy = await navigator.clipboard.writeText(text);
+
+    return textToCopy;
+  }
+
+  const handleCopyClick = async () => {
+    await copyTextToClipboard(copyText);
+    setIsCopied(true);
+    setTimeout(() => {
+      setIsCopied(false);
+    }, 1500);
```

Comment:
There is a very known good practice when using setTimeout. 

- You need to clean the timeout every time the component is unmounted to avoid side effects.
- You may need to store the timeout on a `React.ref` to be able to clean it afterward

You can check more details on this [article](https://felixgerschau.com/react-hooks-settimeout/). Also, you can check a Yoga usage [here](https://github.com/gympass/yoga/blob/master/packages/yoga/src/Snackbar/web/Snackbar.jsx#L115-L125)